mod(coeat): review 기반 기본 CRUD 수정.

LancerAlert opened the pull request:
- 조회 상세 DTO 는 추후 추가.
- #148 , #149 와 충돌할 수 있습니다. 수정 내용에 대한 조율이 필요합니다.

Renseo committed:
mod(coeat): review 기반 기본 CRUD 수정.
- 조회 상세 DTO 는 추후 추가.

mj2sdev committed:
fix(service): coeatService 주석관련 수정

mj2sdev reviewed (approved):
나이쓰 워킹
현재 back/coeat 브랜치에서
storeMapper.existsActive 메소드 빨간 줄 나는데 해당 건만 설명 바랍니다.

LancerAlert commented:
> 나이쓰 워킹 현재 back/coeat 브랜치에서 storeMapper.existsActive 메소드 빨간 줄 나는데 해당 건만 설명 바랍니다.

그고슨 이 `PR` 전에 `StoreMapper` 에 `existsAcive` 가 없었어서 그렇습니다. 이후 별로도 `PR`로 추가하였으니 `develop` 에서는 문제가 없습니다.

mj2sdev committed:
Merge branch 'develop' into back/coeat

Renseo committed:
mod(coeat): id, dto, affected 등 명확하게 수정.
- `id` -> `coeatId`
- `dto` -> `coeat`, `coeatRequest`
- `rootAffected` -> `affectedRoot`
- `coeatAffected` -> `affectedCoeat`
- affected 의 타입을 `int` -> `boolean`

Renseo committed:
mod(coeat): Exception 의 파라미터명 수정.
- `id` -> `coeatId`

Renseo committed:
mod(coeat): coeatMapper.insert 조건 수정.
